tests/optimization/src/TestJs.hx

```diff
@@ -62,7 +62,7 @@ class TestJs {
 		return v + v2;
 	}
 
-	@:js("var a = [];var tmp;try {tmp = a[0];} catch( e ) {tmp = null;}tmp;")
+	@:js("var a = [];var tmp;try {tmp = a[0];} catch( e ) {var e1 = (e instanceof js__$Boot_HaxeError) ? e.val : e;var e2 = e1;tmp = null;}tmp;")
```

Comment:
More generated code because of `@:analyzer(no_local_dce)` is being respected, since this is now inserted as proper AST instead of doing some checks at the generator level.